### PR TITLE
01-chapter1.markdown: grammar and spelling corrections

### PR DESCRIPTION
**Pull request by vlt**
Hi!

While reading progit I made some grammar and spelling corrections.
And to practice I created this first pull request.
Is this the right way to participate?

Daniel

### CODE REVIEW

**Comment by jublo** [comment]
Hello Daniel,

Thank you for your work. We appreciate that you’d like to contribute and make Pro Git better. Yes, sending a pull request is the right way to collaborate!

We’re now proofreading your changes and will update you about the result. Stay tuned!

File: de/01-introduction/01-chapter1.markdown [hunk 1]
```diff
@@ -34,14 +34,14 @@ Bild 1-1. Diagramm: Lokale Versionskontrolle
...
 
-Eines der populärsten Versionskontrollsysteme war rcs, und es wird heute immer noch mit vielen Computern ausgeliefert. Z.B. umfasst auch das Betriebssystem Mac OS X den Befehl rcs, wenn Du die Developer Tools installierst. Dieser Befehl arbeitet nach dem Prinzip, dass er für jede Änderung einen Patch (d.h. eine Kodierung der Unterschiede, die eine Änderung an einer oder mehreren Dateien umfasst) in einem speziellen Format in einer Datei auf der Festplatte speichert.
+Eines der populärsten Versionskontrollsysteme war rcs, und es wird heute immer noch mit vielen Computern ausgeliefert. Zum Beispiel umfasst auch das Betriebssystem Mac OS X den Befehl rcs, wenn Du die Developer-Tools installierst. Dieser Befehl arbeitet nach dem Prinzip, dass er für jede Änderung einen Patch (d. h. eine Kodierung der Unterschiede, die eine Änderung an einer oder mehreren Dateien umfasst) in einem speziellen Format in einer Datei auf der Festplatte speichert.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: `d. h.`: Is this a non-breaking space?

File: de/01-introduction/01-chapter1.markdown [hunk 2]
```diff
@@ -50,18 +50,18 @@ Bild 1-2. Diagramm: Zentralisierte Versionskontrollsysteme
 
 <!--This setup offers many advantages, especially over local VCSs. For example, everyone knows to a certain degree what everyone else on the project is doing. Administrators have fine-grained control over who can do what; and it’s far easier to administer a CVCS than it is to deal with local databases on every client.-->
 
-Dieser Aufbau hat viele Vorteile gegenüber Lokalen Versionskontrollsystemen. Zum Beispiel weiß jeder mehr oder weniger genau darüber Bescheid, was andere, an einem Projekt Beteiligte gerade tun. Administratoren haben die Möglichkeit, detailliert festzulegen, wer was tun kann. Und es ist sehr viel einfacher, ein CVCS zu administrieren als lokale Datenbanken auf jedem einzelnen Anwenderrechner zu verwalten.
+Dieser Aufbau hat viele Vorteile gegenüber Lokalen Versionskontrollsystemen. Zum Beispiel weiß jeder mehr oder weniger genau darüber Bescheid, was andere an einem Projekt beteiligte gerade tun. Administratoren haben die Möglichkeit, detailliert festzulegen, wer was tun kann. Und es ist sehr viel einfacher, ein CVCS zu administrieren als lokale Datenbanken auf jedem einzelnen Anwenderrechner zu verwalten.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: Your second change has incorrect grammar in German. `Beteiligte` is an adjective used as noun, so it needs to be capitalised. The removed comma is correct, though.

File: de/01-introduction/01-chapter1.markdown [hunk 3]
```diff
@@ -50,18 +50,18 @@ Bild 1-2. Diagramm: Zentralisierte Versionskontrollsysteme
...
 
-Und an dieser Stelle kommen verteilte Versionskontrollsysteme (DVCS) ins Spiel. In einem DVCS (wie z.B. Git, Mercurial, Bazaar oder Darcs) erhalten Anwender nicht einfach den jeweils letzten Snapshot des Projektes von einem Server: sie erhalten statt dessen eine vollständige Kopie des Repositories. Auf diese Weise kann, wenn ein Server beschädigt wird, jedes beliebige Repository von jedem beliebigen Anwenderrechner zurück kopiert werden und der Server so wieder hergestellt werden.
+Und an dieser Stelle kommen verteilte Versionskontrollsysteme (DVCS) ins Spiel. In einem DVCS (wie z. B. Git, Mercurial, Bazaar oder Darcs) erhalten Anwender nicht einfach den jeweils letzten Snapshot des Projektes von einem Server: Sie erhalten statt dessen eine vollständige Kopie des Repositorys. Auf diese Weise kann, wenn ein Server beschädigt wird, jedes beliebige Repository von jedem beliebigen Anwenderrechner zurück kopiert und der Server so wieder hergestellt werden.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: - `d. h.`: Is this a non-breaking space?
- Please amend concatenating `statt dessen` to read `stattdessen`.
- Please amend concatenating `zurück kopiert` to read `zurückkopiert`.
- Please amend concatenating `wieder hergestellt` to read `wiederhergestellt`.

File: de/01-introduction/01-chapter1.markdown [hunk 4]
```diff
@@ -70,18 +70,18 @@ Bild 1-3. Diagramm: Distribuierte Versionskontrolle
...
 
-Wie viele großartige Dinge im Leben entstand Git aus kreativem Chaos und hitziger Diskussion. Der Linux Kernel ist ein Open Source Software Projekt von erheblichem Umfang. Während der gesamten Entwicklungszeit des Linux Kernels von 1991 bis 2002 wurden Änderungen an der Software in Form von Patches (d.h. Änderungen an bestehendem Code) und archivierten Dateien herumgereicht. 2002 began man dann, ein proprietäres DVCS System mit dem Namen „Bitkeeper“ zu verwenden.
+Wie viele großartige Dinge im Leben entstand Git aus kreativem Chaos und hitziger Diskussion. Der Linux-Kernel ist ein Open-Source-Softwareprojekt von erheblichem Umfang. Während eines Großteils der Entwicklungszeit des Linux-Kernels (von 1991 bis 2002) wurden Änderungen an der Software in Form von Patches (d. h. Änderungen an bestehendem Code) und archivierten Dateien herumgereicht. 2002 begann man dann, ein proprietäres DVCS namens „Bitkeeper“ zu verwenden.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: `d. h.`: Is this a non-breaking space?

File: de/01-introduction/01-chapter1.markdown [hunk 5]
```diff
@@ -91,20 +91,20 @@ Wie viele großartige Dinge im Leben entstand Git aus kreativem Chaos und hitzig
 
 * Geschwindigkeit
 * Einfaches Design
-* Gute Unterstützung von nicht-linearer Entwicklung (tausende paralleler Branches, d.h. verschiedener Verzweigungen der Versionen)
+* Gute Unterstützung von nicht-linearer Entwicklung (tausende paralleler Branches, d. h. verschiedener Verzweigungen der Versionen)
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 6]
```diff
@@ -120,7 +120,7 @@ Bild 1-4. Andere Systeme speichern Daten als Änderungen an einzelnen Dateien ei
...
+Git sieht Daten nicht in dieser Weise. Stattdessen betrachtet Git seine Daten eher als eine Reihe von Snapshots eines Mini-Dateisystems. Jedes Mal, wenn Du committest (d. h. den gegenwärtigen Status Deines Projektes als eine Version in Git speicherst), sichert Git den Zustand sämtlicher Dateien in diesem Moment („Snapshot“) und speichert eine Referenz auf diesen Snapshot. Um dies möglichst effizient und schnell tun zu können, kopiert Git unveränderte Dateien nicht, sondern legt lediglich eine Verknüpfung zu der vorherigen Version der Datei an. Git betrachtet Daten also wie in Bild 1-5 dargestellt.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 7]
```diff
@@ -129,14 +129,14 @@ Bild 1-5. Git speichert Daten als eine Historie von Snapshots des Projektes.
...
+Dies ist ein wichtiger Unterschied zwischen Git und praktisch allen anderen Versionskontrollsystemen. In Git wurden daher fast alle Aspekte der Versionskontrolle neu überdacht, die in anderen Systemen mehr oder weniger von ihren jeweiligen Vorgängergeneration übernommen worden waren. Git arbeitet im Großen und Ganzen eher wie ein (mit einigen unglaublich mächtigen Werkzeugen ausgerüstetes) Mini-Dateisystem, als wie ein gängiges Versionskontrollsystem. Auf einige der Vorteile, die es mit sich bringt, Daten in dieser Weise zu betrachten, werden wir in Kapitel 3 eingehen, wenn wir das Branchingkonzept von Git diskutieren.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: Consider replacing `mächtigen` with `leistungsstarken`.

File: de/01-introduction/01-chapter1.markdown [hunk 8]
```diff
@@ -144,24 +144,24 @@ Um beispielsweise die Historie des Projektes zu durchsuchen, braucht Git sie nic
...
+Dies bedeutet natürlich außerdem, dass es fast nichts gibt, was Du nicht tun kannst, bloß weil Du gerade offline bist oder keinen Zugriff auf ein VPN hast. Wenn Du im Flugzeug oder Zug ein wenig arbeiten willst, kannst Du problemlos Deine Arbeit committen und Deine Arbeit erst auf den Server pushen (hochladen), wenn Du wieder mit dem Internet verbunden bist. Wenn Du zu Hause bist aber nicht auf das VPN zugreifen kannst, kannst Du dennoch arbeiten. Perforce z. B. erlaubt Dir dagegen nicht sonderlich viel zu tun, solange Du nicht mit dem Server verbunden bist. Und in Subversion und CVS kannst Du Dateien zwar ändern, die Änderungen aber nicht in der Datenbank sichern (weil die Datenbank offline ist). Das mag auf den ersten Blick nicht nach einem großen Problem aussehen, aber Du wirst überrascht sein, was für einen großen Unterschied das ausmachen kann.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: `z. B.`: Is this a non-breaking space?

File: de/01-introduction/01-chapter1.markdown [hunk 9]
```diff
@@ -144,24 +144,24 @@ Um beispielsweise die Historie des Projektes zu durchsuchen, braucht Git sie nic
...
 
-In Git werden Änderungen in Checksummen umgerechnet, bevor sie gespeichert werden. Anschließend werden sie mit dieser Checksumme referenziert. Das macht es unmöglich, dass sich die Inhalte von Dateien oder Verzeichnissen ändern, ohne dass Git das mitbekommt. Git basiert auf dieser Funktionalität und sie ist ein integraler Teil von Gits Philosophie. Man kann Informationen deshalb z.B. nicht während der Übermittlung verlieren oder unwissentlich beschädigte Dateien verwenden, ohne dass Git in der Lage wäre, dies festzustellen.
+In Git werden Änderungen in Checksummen umgerechnet, bevor sie gespeichert werden. Anschließend werden sie mit dieser Checksumme referenziert. Das macht es unmöglich, dass sich die Inhalte von Dateien oder Verzeichnissen ändern, ohne dass Git das mitbekommt. Git basiert auf dieser Funktionalität und sie ist ein integraler Teil von Gits Philosophie. Man kann Informationen deshalb z. B. nicht während der Übermittlung verlieren oder unwissentlich beschädigte Dateien verwenden, ohne dass Git in der Lage wäre, dies festzustellen.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 10]
```diff
@@ -212,11 +212,11 @@ Der grundlegend Git Arbeitsprozess sieht in etwa so aus:
 
 1. Du bearbeitest Dateien in Deinem Arbeitsverzeichnis.
 2. Du markierst Dateien für den nächsten Commit, indem Du Snapshots zur Staging Area hinzufügst.
-3. Du legst den Commit an, wodurch die in der Staging Area vorgemerkten Snapshots dauerhaft im Git Verzeichnis (d.h. der lokalen Datenbank) gespeichert werden.
+3. Du legst den Commit an, wodurch die in der Staging Area vorgemerkten Snapshots dauerhaft im Git-Verzeichnis (d. h. der lokalen Datenbank) gespeichert werden.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 11]
```diff
@@ -230,11 +230,11 @@ Lass uns damit anfangen, Git tatsächlich zu verwenden. Der erste Schritt besteh
...
 
-Wenn es Dir möglich ist, empfehlen wir, Git vom Quellcode aus zu installieren, weil Du die jeweils neueste Version erhältst. In der Regel bringt jede Version nützliche Verbesserungen (z.B. am Interface), sodass es sich lohnt die jeweils neueste Version zu verwenden – sofern Du natürlich damit klarkommst, Software aus dem Quellcode zu kompilieren. Viele Linux Distributionen umfassen sehr alte Git Versionen. Wenn Du also keine sehr aktuelle Distribution oder Backports (xxx) verwendest, empfehlen wir, diesen Weg in Erwägung ziehen.
+Wenn es Dir möglich ist, empfehlen wir, Git vom Quellcode aus zu installieren, weil Du die jeweils neueste Version erhältst. In der Regel bringt jede Version nützliche Verbesserungen (z. B. am Interface), sodass es sich lohnt, die jeweils neueste Version zu verwenden – sofern Du natürlich damit klarkommst, Software aus dem Quellcode zu kompilieren. Viele Linux-Distributionen umfassen sehr alte Git-Versionen. Wenn Du also keine sehr aktuelle Distribution oder Backports (xxx) verwendest, empfehlen wir, diesen Weg in Erwägung ziehen.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 12]
```diff
@@ -230,11 +230,11 @@ Lass uns damit anfangen, Git tatsächlich zu verwenden. Der erste Schritt besteh
...
 
 <!--To install Git, you need to have the following libraries that Git depends on: curl, zlib, openssl, expat, and libiconv. For example, if you’re on a system that has yum (such as Fedora) or apt-get (such as a Debian based system), you can use one of these commands to install all of the dependencies:-->
 
-Um Git zu installieren, benötigst Du die folgenden Bibliotheken, die von Git verwendet werden: curl, zlib, openssl, expat und libiconv. Wenn Dir auf Deinem System yum (z.B. auf Fedora) oder apt-get (z.B. auf Debian-basierten Systemen) zur Verfügung steht, kannst Du einen der folgenden Befehle verwenden, um diese Abhängigkeiten zu installieren:
+Um Git zu installieren, benötigst Du die folgenden Bibliotheken, die von Git verwendet werden: curl, zlib, openssl, expat und libiconv. Wenn Dir auf Deinem System yum (z. B. auf Fedora) oder apt-get (z. B. auf Debian-basierten Systemen) zur Verfügung steht, kannst Du einen der folgenden Befehle verwenden, um diese Abhängigkeiten zu installieren:
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 13]
```diff
@@ -244,7 +244,7 @@ Um Git zu installieren, benötigst Du die folgenden Bibliotheken, die von Git ve
 
 <!--When you have all the necessary dependencies, you can go ahead and grab the latest snapshot from the Git web site:-->
 
-Nachdem Du die genannten Bibliotheken installiert hast, besorge Dir die aktuelle Version des Git Quellcodes von der Git Webseite:
+Nachdem Du die genannten Bibliotheken installiert hast, besorg Dir die aktuelle Version des Git-Quellcodes von der Git-Webseite:
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: Consider changing `Webseite` to `Website`.

File: de/01-introduction/01-chapter1.markdown [hunk 14]
```diff
@@ -283,7 +283,7 @@ Auf einem Debian-basierten System wie Ubuntu steht Dir apt-get zur Verfügung:
 
 <!--There are two easy ways to install Git on a Mac. The easiest is to use the graphical Git installer, which you can download from the SourceForge page (see Figure 1-7):-->
 
-Auf einem Mac kann man Git auf zwei Arten installieren. Der einfachste ist, das grafische Git Installationsprogramm zu verwenden, den man von der SourceForge Webseite herunterladen kann (siehe Bild 1-7)
+Auf einem Mac kann man Git auf zwei Arten installieren. Die einfachste ist, das grafische Git-Installationsprogramm zu verwenden, das man von der SourceForge-Webseite herunterladen kann (siehe Bild 1-7)
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: Consider appending a trailing colon `:` like in the source.

File: de/01-introduction/01-chapter1.markdown [hunk 15]
```diff
@@ -317,17 +317,17 @@ Alternativ kann man Git auch über Homebrew (`http://brew.sh/`) installieren. Ha
 
 <!--Installing Git on Windows is very easy. The msysGit project has one of the easier installation procedures. Simply download the installer exe file from the GitHub page, and run it:-->
 
-Das msysGit Projekt macht die Installation von Git unter Windows sehr einfach. Lade einfach das Installationsprogramm für Windows von der GitHub Webseite herunter und führe es aus:
+Das msysGit Projekt macht die Installation von Git unter Windows sehr einfach. Lad einfach das Installationsprogramm für Windows von der GitHub-Webseite herunter und führe es aus:
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: This imperative has incorrect grammar in German. Verben mit Verbstammendung: `-t`, `-d`, `-ig`, `-er`, `-el` bekommen im Imperativ die Endung `-e`.

File: de/01-introduction/01-chapter1.markdown [hunk 16]
```diff
@@ -346,54 +346,54 @@ Git umfasst das Werkzeug `git config`, das Dir erlaubt, Konfigurationswerte zu v
...
 
 <!--### Your Editor ###-->
 ### Dein Editor ###
 
 <!--Now that your identity is set up, you can configure the default text editor that will be used when Git needs you to type in a message. By default, Git uses your system’s default editor, which is generally Vi or Vim. If you want to use a different text editor, such as Emacs, you can do the following:-->
 
-Nachdem Du Deine Identität jetzt konfiguriert hast, kannst Du einstellen, welchen Texteditor Git in Situationen verwenden soll, in denen Du eine Nachricht eingeben musst. Normalerweise verwendet Git den Standard-Texteditor Deines Systems – das ist üblicherweise Vi oder Vim. Wenn Du einen anderen Texteditor, z.B. Emacs, verwenden willst, kannst Du das wie folgt festlegen:
+Nachdem Du Deine Identität jetzt konfiguriert hast, kannst Du einstellen, welchen Texteditor Git in Situationen verwenden soll, in denen Du eine Nachricht eingeben musst. Normalerweise verwendet Git den Standard-Texteditor Deines Systems – das ist üblicherweise Vi oder Vim. Wenn Du einen anderen Texteditor, z. B. Emacs, verwenden willst, kannst Du das wie folgt festlegen:
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 17]
```diff
@@ -346,54 +346,54 @@ Git umfasst das Werkzeug `git config`, das Dir erlaubt, Konfigurationswerte zu v
...
-Git kann von Hause aus mit den folgenden Diff Programmen arbeiten: kdiff3, tkdiff, meld, xxdiff, emerge, vimdiff, gvimdiff, ecmerge, and opendiff. Außerdem kannst Du ein eigenes Programm aufsetzen. Wir werden in Kapitel 7 darauf eingehen, wie das geht.
+Git kann von Hause aus mit den folgenden Diff-Programmen arbeiten: kdiff3, tkdiff, meld, xxdiff, emerge, vimdiff, gvimdiff, ecmerge, and opendiff. Außerdem kannst Du ein eigenes Programm aufsetzen. Wir werden in Kapitel 7 darauf eingehen, wie das geht.
 
 <!--### Checking Your Settings ###-->
 ### Deine Einstellungen überprüfen ###
 
 <!--If you want to check your settings, you can use the `git config -\-list` command to list all the settings Git can find at that point:-->
 
-Wenn Du Deine Einstellungen überprüfen willst, kannst Du mit dem Befehl `git config --list` alle Einstellungen anzuzeigen, die Git an dieser Stelle (z.B. innerhalb eines bestimmten Projektes) bekannt sind:
+Wenn Du Deine Einstellungen überprüfen willst, kannst Du mit dem Befehl `git config --list` alle Einstellungen anzuzeigen, die Git an dieser Stelle (z. B. innerhalb eines bestimmten Projektes) bekannt sind:
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 18]
```diff
@@ -406,7 +406,7 @@ Wenn Du Deine Einstellungen überprüfen willst, kannst Du mit dem Befehl `git c
 
 <!--You may see keys more than once, because Git reads the same key from different files (`/etc/gitconfig` and `~/.gitconfig`, for example). In this case, Git uses the last value for each unique key it sees.-->
 
-Manche Variablen werden möglicherweise mehrfach aufgelistet, weil Git dieselbe Variable in verschiedenen Dateien (z.B. `/etc/gitconfig` und `~/.gitconfig`) findet. In diesem Fall verwendet Git dann den jeweils zuletzt aufgelisteten Wert.
+Manche Variablen werden möglicherweise mehrfach aufgelistet, weil Git dieselbe Variable in verschiedenen Dateien (z. B. `/etc/gitconfig` und `~/.gitconfig`) findet. In diesem Fall verwendet Git dann den jeweils zuletzt aufgelisteten Wert.
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: See above.

File: de/01-introduction/01-chapter1.markdown [hunk 19]
```diff
@@ -416,11 +416,11 @@ Außerdem kannst Du mit dem Befehl `git config {key}` prüfen, welchen Wert Git
 	Scott Chacon
 
 <!--## Getting Help ##-->
-## Hilfe finden ##
+## Hilfe suchen ##
 
 <!--If you ever need help while using Git, there are three ways to get the manual page (manpage) help for any of the Git commands:-->
 
-Falls Du jemals Hilfe in der Anwendung von Git benötigst, gibt es drei Möglichkeiten, die entsprechende Seite aus der Dokumentation (manpage) für jeden Git Befehl anzuzeigen:
+Falls Du jemals Hilfe bei der Anwendung von Git benötigst, gibt es drei Möglichkeiten, die entsprechende Seite aus der Dokumentation (manpage) für jeden Git Befehl anzuzeigen:
```
Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: Think about whether we might use `Nutzung` instead of `Anwendung` here?

Comment (jublo) [inline de/01-introduction/01-chapter1.markdown]: How about inserting a dash between `Git Befehl` (globally)?

**Comment by jublo** [comment]
@vlt My compliment for a well-engineered review of chapter 1. Thank you for your hard work! Please find my comments inline.

**Comment by vlt** [comment]
@jublo 
Hello!

Thank you for reviewing and adding to my changes in chapter 1.

I tried to correct my previous mistakes with the recent commits.

**Comment by vlt** [comment]
As a side note:
While continuing reading on the weekend I discovered that there’s progit2.

This is mainly an exercise in git for me but do you think it would make more sense to continue with progit2?

Daniel

**Pull request merged**